### ISSUE TITLE
Added Support for Hebrew

### ISSUE THREAD

**Issue by bar-boko** [emordonez/connecting-wall#6]
It turns out that you don't support Hebrew in your app, because of the transform to base64 for the link. It doesn't work property with Unicode lauguages.

I found a way to fix it very easily.
I'd like to get a permission to create a branch to add this fix.

**Comment by emordonez** [comment]
Yes, by all means! Thank you for noticing that and bringing it to my attention. Unicode support hadn't occurred to me since the base64 encoding was meant to be a temporary solution until I could find a better way to save walls, but then I forgot about this project, of course :/
